requirements.pgstac.txt

```diff
@@ -7,3 +7,5 @@ buildpg
 brotli_asgi
 pygeofilter>=0.2
 pypgstac>=0.8,<0.10
+python-jose>=3.3.0
+python-multipart
```

Comment:
esto para qué?

Reply:
python-jose>=3.3.0
Proporciona herramientas para manejar JSON Web Tokens (JWT) y realizar operaciones asociadas al estándar JOSE (JSON Object Signing and Encryption)

y  python-multipart es usada por la dependencia OAuth2PasswordRequestForm que requiere que los datos se envíen como multipart/form-data

Reply:
esa dependencia (OAuth2PasswordRequestForm) solo se está usando para tipar un parámetro. Realmente es necesario importar `python-multipart` para esto? si no se agrega qué error sale?

Reply:
> esa dependencia (OAuth2PasswordRequestForm) solo se está usando para tipar un parámetro. Realmente es necesario importar `python-multipart` para esto? si no se agrega qué error sale?

aparece este error:Form data requires "python-multipart" to be installed. 
2024-12-03 11:27:18 You can install "python-multipart" with: 
2024-12-03 11:27:18 
2024-12-03 11:27:18 pip install python-multipart
2024-12-03 11:27:18 
2024-12-03 11:27:18 Traceback (most recent call last):
2024-12-03 11:27:18   File "<frozen runpy>", line 198, in _run_module_as_main
2024-12-03 11:27:18   File "<frozen runpy>", line 88, in _run_code
2024-12-03 11:27:18   File "/usr/local/lib/python3.11/site-packages/stac_fastapi/pgstac/app.py", line 12, in <module>
2024-12-03 11:27:18     from stac_fastapi.api.app import StacApi
2024-12-03 11:27:18   File "/usr/local/lib/python3.11/site-packages/stac_fastapi/api/app.py", line 34, in <module>
2024-12-03 11:27:18     from stac_fastapi.api.auth import auth_router
2024-12-03 11:27:18   File "/usr/local/lib/python3.11/site-packages/stac_fastapi/api/auth.py", line 14, in <module>
2024-12-03 11:27:18     @auth_router.post("/token")

Reply:
bueno, ni modo